config.sample.yaml

```diff
@@ -111,7 +111,6 @@ services:
 
   - ID: "slackapi_service"
     Type: "slackapi"
-    UserID: "@slackapi:localhost" # requires a Syncing client
```

Comment:
Go-NEB has no concept of personal or global bots. It's just a user who sends messages. You need to give a user ID.